src/components/Header/Header.tsx

```diff
@@ -19,17 +19,16 @@ export const Header = ({ walletProps: { state, address, name,
   expectedNetworkName, connect,
   switchNetwork, register,
   switchAccount, openExplorer }, currency, balance }: headerProps) => {
-  const [isOpenDropdown, setIsOpenDropdown] = useState<boolean>(false)
 
   const menuList = [
     {
       title: 'About',
-      href: routes.index,
+      href: 'https://plush.family/',
```

Comment:
How these changes relates to the task https://plush.atlassian.net/browse/PF-203 ?